packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -56,6 +59,34 @@ class NetworkController extends DisposableController
     );
     subscribeToFilterChanges();
   }
+  List<DartIOHttpRequestData>? _httpRequests;
+
+  String? exportAsHarFile() {
+    final exportController = ExportController();
+
+    _httpRequests =
+        filteredData.value.whereType<DartIOHttpRequestData>().toList();
+
+    if (_httpRequests!.isEmpty) {
+      debugPrint('No valid request data to export');
+      return '';
+    }
+
+    try {
+      if (_httpRequests != null && _httpRequests!.isNotEmpty) {
```

Comment:
if you check `.isNullOrEmpty` above (see previous comment), you don't need to check this condition at all here and can remove the if statement

Reply:
yes right, removed this